script/zlib.decompress.py

```diff
@@ -0,0 +1,20 @@
+import zlib
+import sys
+
+CHUNKSIZE = 1024
+zlib_obj = zlib.decompressobj(int(sys.argv[2]))
+compressed_file = open(sys.argv[1], 'rb')
+buf = compressed_file.read(CHUNKSIZE)
+
+decompressed_file = open(sys.argv[3], 'wb')
+
+# Decompress stream chunks
+while buf:
+    decompressed_data = zlib_obj.decompress(buf)
+    buf = compressed_file.read(CHUNKSIZE)
+    decompressed_file.write(decompressed_data)
+
+decompressed_file.write(zlib_obj.flush())
```

Comment:
obal to nejakym try-catchem, v soucasne verzi to vyhodi stack trace, kterej je uzivateli k prdu:
```
Failed decompressing gzip file /data/in/files/test_zlib: The command "'python3' '/code/script/zlib.decompress.py' '/data/in/files/test_zlib' '15' '/data/out/files/test_zlib/test_zlib'" failed.  Exit Code: 1(General error)  Working directory: /code  Output: ================   Error Output: ================ Traceback (most recent call last):   File "/code/script/zlib.decompress.py", line 13, in <module>     decompressed_data = zlib_obj.decompress(buf) zlib.error: Error -3 while decompressing data: incorrect header check
```

Reply:
Dal sem tam try expect, každopádně je to pro mě španělská vesnice, nevím jestli z té error message ide vytáhnút něco užitečného, protože by to mohl failnůt na souboru, ale taky može byt špatně ten window_size

Reply:
ne, to je v pohode, ze to napise " Error -3 while decompressing data: incorrect header check" to uz jako nic lepsiho nevymyslime, jde jen o to aby to neblilo ten strack trace

Reply:
OK upravil sem tu error msg